lib/lib_core.dart

```diff
@@ -34,4 +37,28 @@ class FlutterBleLib {
     await _methodChannel.invokeMethod(MethodName.DESTROY_CLIENT);
     return;
   }
+
+  Stream<ScanResult> startDeviceScan(int scanMode,
+      int callbackType,
+      List<String> uuids,) async* {
+    await _methodChannel.invokeMethod(
```

Comment:
I think we might lose a few first results, if they come in instantaneously? We're waiting for the future to resolve before we subscribe to the event channel.

Reply:
Fixed.